src/fork_gc.h

```diff
@@ -18,11 +18,21 @@ typedef struct {
   uint64_t gcBlocksDenied;
 } ForkGCStats;
 
+typedef enum ForkGCCtxType {
```

Comment:
If this enum isn't used anywhere except within forkgc; then this should just be in the .c file

Reply:
its been used by ForkGCCtx which is also defined on this h file is is used outside of it.